new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleDomainValidation.java

```diff
@@ -41,6 +49,49 @@ public void testCreatingDomain() {
         "Failed to create unique namespace due to ApiException");
     logger.info("Got a new namespace called {0}", namespace);
 
+    // create persistent volume and persistent volume claim
+    final String pvcName = domainUID + "-pvc"; // name of the persistent volume claim
+    final String pvName = domainUID + "-pv"; // name of the persistent volume
+
+    V1PersistentVolumeClaim v1pvc = new V1PersistentVolumeClaim()
+        .spec(new V1PersistentVolumeClaimSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(domainUID + "-weblogic-domain-storage-class")
+            .volumeName(domainUID + "-weblogic-pv")
+            .resources(new V1ResourceRequirements()
+                .putRequestsItem("storage", Quantity.fromString("10Gi"))))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvcName)
+            .withNamespace(namespace)
+            .build()
+            .putLabelsItem("weblogic.resourceVersion", "domain-v2")
+            .putLabelsItem("weblogic.domainUID", domainUID));
+
+    boolean success = assertDoesNotThrow(
+        () -> TestActions.createPersistentVolumeClaim(v1pvc)
+    );
+    assertTrue(success, "PersistentVolumeClaim creation failed");
+
+    V1PersistentVolume v1pv = new V1PersistentVolume()
+        .spec(new V1PersistentVolumeSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(domainUID + "-weblogic-domain-storage-class")
+            .volumeMode("Filesystem")
+            .putCapacityItem("storage", Quantity.fromString("10Gi"))
+            .persistentVolumeReclaimPolicy("Recycle")
+            .hostPath(new V1HostPathVolumeSource()
+                .path(System.getProperty("java.io.tmpdir") + domainUID + "-persistentVolume")))
```

Comment:
Is this location guaranteed to be shared by all hosts in the k8s cluster?

Reply:
It doesn't seem likely.

Reply:
Good point... I completely missed the intent while looking at the structure.

Reply:
Yes it is not shared. This whole PV creation is meant to be an example on how to create it. This has to be replaced with whatever shared location we should be using in the actual cluster.